src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -185,7 +186,22 @@ private AuthCredentials extractCredentials0(final SecurityRequest request) {
                 final AuthCredentials ac = new AuthCredentials(subject, roles).markComplete();
 
                 for (Entry<String, Object> claim : claims.entrySet()) {
-                    ac.addAttribute("attr.jwt." + claim.getKey(), String.valueOf(claim.getValue()));
+                    String key = "attr.jwt." + claim.getKey();
+                    Object value = claim.getValue();
```

Comment:
Are there are edge cases where the collection might be empty, contain null values, or have un expected types? Do we need to do some validations?

Reply:
I added an additional test, it will produce the same claims list from the JWT and ensure that its proper JSON (meaning that the strings will be enclosed with double quotes)